app/src/main/java/com/example/cyclofit/ui/utils/ChartUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.cyclofit.ui.utils
+
+import com.github.mikephil.charting.utils.ColorTemplate
+
+object ChartUtils {
+
+    // add a lot of colors
+    val colors = ArrayList<Int>().apply {
+        for (c in ColorTemplate.VORDIPLOM_COLORS) add(c)
+        for (c in ColorTemplate.JOYFUL_COLORS) add(c)
+        for (c in ColorTemplate.COLORFUL_COLORS) add(c)
+        for (c in ColorTemplate.LIBERTY_COLORS) add(c)
+        for (c in ColorTemplate.PASTEL_COLORS) add(c)
+        add(ColorTemplate.getHoloBlue())
+    }
+
+    val months = arrayOf(
+        "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Okt", "Nov", "Dec"
+    )
```

Comment:
Use this as enums instead of an array. Also make sure that you define them in a separate file. So basically make and Enum by the name of `Months.kt` and define months there